linux/powershell/setupPowerShell.ps1

```diff
@@ -142,6 +142,8 @@ try {
         PowerShellGet\Install-Module -Name ExchangeOnlineManagement -RequiredVersion 2.0.5 -Force
         PowerShellGet\Install-Module -Name Microsoft.PowerShell.SecretManagement @prodAllUsers
         PowerShellGet\Install-Module -Name Microsoft.PowerShell.SecretStore @prodAllUsers
+        PowerShellGet\Install-Module -Name PSReadLine -Repository PSGallery
+        PowerShellGet\Install-Module -Name Az.Accounts -Repository PSGallery
```

Comment:
PSReadLine and Az.Accounts modules are already present.

Reply:
I needed to ensure that new versions where installed and I didn't see them listed.